### PR TITLE
More Food

### PR DESCRIPTION
**Pull request by BurgerLUA**
# Overview
Adds more food for the chef to cook. Items include:
Bacon Flatbread (Not pizza)
Breakfast Wrap
Meat Pocket
Fish Taco
Pineapple Rings
Pineapple Pizza
Chocolate Truffles
Bacon Burger
NtMuffin
BLT

## Icons

Food

### CODE REVIEW

File: code/game/objects/items/weapons/storage/fancy.dm [hunk 1]
```diff
@@ -295,3 +295,18 @@
...
 	update_icon()
+
+/obj/item/weapon/storage/fancy/chocolate_box
+	icon = 'icons/obj/chocolate.dmi'
+	icon_state = "chocolatebox"
+	icon_type = "chocolatebox"
+	name = "chocolate box"
+	storage_slots = 8
+	can_hold = list(
+		/obj/item/weapon/reagent_containers/food/snacks,
```
Comment (Lohikar) [inline code/game/objects/items/weapons/storage/fancy.dm]: Trailing comma.

File: code/game/objects/items/weapons/storage/misc.dm [hunk 2]
```diff
@@ -49,3 +49,19 @@
...
+	var/startswith = 6
+	can_hold = list(/obj/item/weapon/reagent_containers/food/snacks/pineapple_ring)
+
+/obj/item/weapon/storage/box/pineapple/fill()
+	for(var/i=1; i <= startswith; i++)
+		new /obj/item/weapon/reagent_containers/food/snacks/pineapple_ring(src)
+	update_icon()
+	
+	
+
```
Comment (Lohikar) [inline code/game/objects/items/weapons/storage/misc.dm]: What's with all this whitespace?

**Comment by Kaedwuff** [comment]
No ham/bacon on the pineapple pizza? What's wrong with you D:<

File: code/game/objects/items/weapons/storage/misc.dm (hunk 2 above)
Comment (BurgerLUA) [reply code/game/objects/items/weapons/storage/misc.dm]: Bad habbit. I'll remove it.

**Comment by BurgerLUA** [comment]
god damn it arrow you broke everything

**Pull request merged**